cura/PrinterOutputDevice.py

```diff
@@ -57,6 +61,39 @@ def requestWrite(self, node, file_name = None, filter_by_machine = False):
     # it also sends it's own device_id (for convenience sake)
     connectionStateChanged = pyqtSignal(str)
 
+    timeElapsedChanged = pyqtSignal()
+
+    timeTotalChanged = pyqtSignal()
+
+    jobStateChanged = pyqtSignal()
+
+    jobNameChanged = pyqtSignal()
```

Comment:
I personally prefer to put the signal next to the property it is a change notification of, since that requires less searching. But we have no hard requirement for it.